analysis/ana_log.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+class Ana {
+  static success(counterPath) {
+    var path = counterPath + "/success";
+    if (arguments.length > 1) {
+      console.log(path, ...Array.prototype.slice.call(arguments, 1));
+    } else {
+      console.log(path);
+    }
+  }
+  static fail(counterPath) {
+    var path = counterPath + "/fail";
+    if (arguments.length > 1) {
+      console.log(path, ...Array.prototype.slice.call(arguments, 1));
+    } else {
+      console.log(path);
+    }
+  }
+  static log() {
+    console.log(...arguments);
+  }
+}
+
+module.exports = {
+  Ana: Ana
```

Comment:
Although `module.exports = Ana;` is probably all you need, unless you expect to export something else later?